paymentsheet/src/main/java/com/stripe/android/paymentsheet/model/PaymentOptionFactory.kt

```diff
@@ -24,14 +26,18 @@ internal class PaymentOptionFactory @Inject constructor(
             Configuration.UI_MODE_NIGHT_YES
     }
 
-    private suspend fun loadPaymentOption(paymentOption: PaymentOption): Drawable {
+    @VisibleForTesting
+    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+    suspend fun loadPaymentOption(paymentOption: PaymentOption): Drawable {
```

Comment:
Does this still need to be non private? I guess it does. The other thing we could do is call paymentOptionFactory.create(), so that we use the internal logic for it instead. Not sure if it's worth though.

Reply:
yeah, I thought about that. The issue is that I want to test specific combos of icon url + icon res and I want it to be explicit what those combinations are. If I wrote the test based on the output of PaymentOptionFactory.create(), I think it'd be harder to understand what the intended test set up is. 

I think you could definitely make an argument that we should test this via our public functions and instead write tests based on specific PaymentSelection types. But imo a test that e.g. verifies that a PaymentSelection.GooglePay uses a resource icon is actually less useful, because if we updated that to use an icon url in the future that would be fine. I don't feel super strongly either way.